ads/navegg.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+import {doubleclick} from '../ads/google/doubleclick';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function navegg(global, data) {
+  validateData(data, ['acc']);
+  const acc = data.acc;
+  let seg, nvg = function() {};
+  delete data.acc;
+  nvg.prototype.getProfile = function() {};
+  loadScript(global, 'https://tag.navdmp.com/amp.1.0.0.min.js', () => {
+    nvg = global[`nvg${acc}`] = new global['AMPNavegg']({
+      acc,
+    });
+    nvg.getProfile(nvgTargeting => {
+      for (seg in nvgTargeting) {
+        data.targeting[seg] = nvgTargeting[seg];
```

Comment:
We intend to be an extension to the doubleclick Ads, all of these additional parameters coming from the doubleclick documentation. We append a line in the md file explaining that.